src/mapper.rs

```diff
@@ -205,17 +205,20 @@ impl<'s> ProguardMapper<'s> {
     /// # Example
     ///
     /// ```
-    /// use proguard::{ProguardMapper,Throwable};
+    /// use proguard::{ProguardMapper, Throwable};
     ///
     /// let mapping = "com.example.Mapper -> a.b:";
     /// let mapper = ProguardMapper::from(mapping);
     ///
     /// let throwable = Throwable::try_parse(b"a.b: Crash").unwrap();
     /// let mapped = mapper.remap_throwable(&throwable);
     ///
-    /// assert_eq!(Some(Throwable::with_message("com.example.Mapper", "Crash")), mapped);
+    /// assert_eq!(
+    ///     Some(Throwable::with_message("com.example.Mapper", "Crash")),
+    ///     mapped
+    /// );
     /// ```
-    pub fn remap_throwable<'a>(&'a self, throwable: &Throwable<'a>) -> Option<Throwable<'a>> {
+    pub fn remap_throwable(&'s self, throwable: &Throwable<'s>) -> Option<Throwable<'s>> {
```

Comment:
Are you sure this is equivalent? Previously, this was bound to the lifetime of `self` and of the data in `Throwable`, Now, this is bound to `'s`, and you're requiring that the self lifetime is equal to `'s`. Can you double-check lifetime bounds here, please?